app/src/main/java/sopt/haeti/baeminaos/presentation/cart/CartActivity.kt

```diff
@@ -1,13 +1,52 @@
 package sopt.haeti.baeminaos.presentation.cart
 
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
 import sopt.haeti.baeminaos.R
-import sopt.haeti.baeminaos.databinding.ActivityHomeBinding
+import sopt.haeti.baeminaos.databinding.ActivityCartBinding
 import sopt.haeti.baeminaos.util.base.BindingActivity
+import timber.log.Timber
 
-class CartActivity : BindingActivity<ActivityHomeBinding>(R.layout.activity_home) {
+class CartActivity : BindingActivity<ActivityCartBinding>(R.layout.activity_cart) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(binding.root)
+
+        Timber.plant(Timber.DebugTree())
+
+        // 툴바 설정
+        setSupportActionBar(binding.toolbarCart)
+        supportActionBar?.setDisplayShowTitleEnabled(false)
+        // 툴바의 뒤로가기 버튼 설정
+        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
+        supportActionBar!!.setHomeAsUpIndicator(R.drawable.ic_back)
+
+        // 이미지뷰 테두리 설정
+        binding.ivCartItem1Title.clipToOutline = true
+        binding.ivCartItem1Food.clipToOutline = true
+        binding.ivCartItem2Title.clipToOutline = true
+        binding.ivCartItem2Food.clipToOutline = true
```

Comment:
라이브러리를 안쓰고 구현하셨군요! ShapeableImageView나 CirecleImageView보다 이 방법이 좋을까요? 궁금하네요 ㅎ
Kotlin Scope Function 중 with 를 사용해서 간단하게 바꿀 수 있습니다. 그리고 함수화 꼭 해주세요!!
```suggestion
        // 이미지뷰 테두리 설정
        with(binding) {
                ivCartItem1Title.clipToOutline = true
               ivCartItem1Food.clipToOutline = true
               ivCartItem2Title.clipToOutline = true
               ivCartItem2Food.clipToOutline = true
        }
```